KubeArmor/feeder/policyMatcher.go

```diff
@@ -1008,18 +1008,58 @@ func (fd *Feeder) UpdateMatchedPolicy(log tp.Log) tp.Log {
 				}
 			}
 
-			if log.ProcessVisibilityEnabled && log.Operation == "Process" {
-				log.Type = "ContainerLog"
-				return log
-			} else if log.FileVisibilityEnabled && log.Operation == "File" {
-				log.Type = "ContainerLog"
-				return log
-			} else if log.NetworkVisibilityEnabled && log.Operation == "Network" {
-				log.Type = "ContainerLog"
-				return log
-			} else if log.CapabilitiesVisibilityEnabled && log.Operation == "Capabilities" {
-				log.Type = "ContainerLog"
-				return log
+			if log.Operation == "Process" {
+				if cfg.GlobalCfg.DefaultFileArmor == "block" {
+					log.Type = "DefaultArmor"
+					log.Action = "ImplicitBlock"
+					return log
+				} else if cfg.GlobalCfg.DefaultFileArmor == "audit" {
+					log.Type = "DefaultArmor"
+					log.Action = "ImplicitAudit"
+					return log
+				} else if log.ProcessVisibilityEnabled {
+					log.Type = "ContainerLog"
+					return log
+				}
+			} else if log.Operation == "File" {
+				if cfg.GlobalCfg.DefaultFileArmor == "block" {
+					log.Type = "DefaultArmor"
+					log.Action = "ImplicitBlock"
+					return log
+				} else if cfg.GlobalCfg.DefaultFileArmor == "audit" {
+					log.Type = "DefaultArmor"
+					log.Action = "ImplicitAudit"
+					return log
+				} else if log.FileVisibilityEnabled {
+					log.Type = "ContainerLog"
+					return log
+				}
+			} else if log.Operation == "Network" {
+				if cfg.GlobalCfg.DefaultNetworkArmor == "block" {
```

Comment:
This code looks redundant. Can we wrap this in function and have something like this?
```golang
func setLogFields(action string, log *types.Log) {
     if cfg.GlobalCfg.DefaultCapabilityArmor == "block" {
	(*log).Type = "DefaultArmor"
	(*log).Action = "ImplicitBlock"
     } else if cfg.GlobalCfg.DefaultCapabilityArmor == "audit" {
	  (*log).Type = "DefaultArmor"
	  (*log).Action = "ImplicitAudit"
     } else if (*log).CapabilitiesVisibilityEnabled {
	  (*log).Type = "ContainerLog"
     }
}

setLogFields(cfg.GlobalCfg.DefaultCapabilityArmor, &log)
```